src/app/token/[tokenId]/getTokenInfo.ts

```diff
@@ -44,14 +45,23 @@ async function getBasicTokenInfo(
       throw new Error('Unable to fetch token info for this request');
     }
 
-    console.log('[debug] cache miss, fetching ', `${apiUrl}/metadata/v1/ft/${tokenId}`);
+    // const contractInfoResponse = await fetch(`${apiUrl}/extended/v1/contract/${tokenId}`);
```

Comment:
This is the code to get the circulating supply from the holders endpoint. I am commenting it out as discussed so the data doesn't show next to the total supply and potenitally show an inconsistency/inaccuracy